presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -855,13 +863,13 @@ public PlanOptimizers(
         // Optimizers above this do not need to care about aggregations with the type other than SINGLE
         // This optimizer must be run after all exchange-related optimizers
         builder.add(new IterativeOptimizer(
-                metadata,
```

Comment:
Unrelated change?

Reply:
IntelliJ formatter didn't like this indentation for some reason. Fixed